packages/flutter/lib/src/cupertino/activity_indicator.dart

```diff
@@ -128,13 +143,17 @@ class _CupertinoActivityIndicatorPainter extends CustomPainter {
     canvas.save();
     canvas.translate(size.width / 2.0, size.height / 2.0);
 
+    // The standard iOS implementation has the top tick appearing first,
+    // so need to rotate so that that is the first one that gets drawn
```

Comment:
Needs a period at the end of the sentence.  If you want you could probably even simplify this to:

```
// Rotate so that the top tick is the first to be drawn.
```

Reply:
Fixed.